doc/API/Devices.md

```diff
@@ -1264,6 +1264,51 @@ Output:
 ]
 ```
 
+### `update_device_attrib`
+
+Update devices attributes in the database.
+
+Route: `/api/v0/attribs/:hostname`
+
+- hostname can be either the device hostname or id
+
+Input (JSON):
+
+- attrib_type: The attribute type within the database (can be an array of fields)
+- attrib_value: The attribute value to update the column with (can be an array of data))
```

Comment:
Why not use key value pairs instead of two separate arrays?

Reply:
The device api uses a field: name, data: value format.  I figured it was best to stay in the style of the existing api.